src/inverseRobot.js

```diff
@@ -24,7 +24,31 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const inversed = {};
+
+  const arrKeys = Object.keys(robot);
+
+  if (!findDuplicates(arrKeys)) {
+    return null;
+  }
+
+  const arrValues = Object.values(robot);
+
+  if (!findDuplicates(arrValues)) {
+    return null;
+  }
+
+  for (const key in robot) {
+    inversed[robot[key]] = key;
+  }
+
+  return inversed;
+}
+
+function findDuplicates(array) {
+  const noDuplicatesArray = Array.from(new Set(array));
+
+  return noDuplicatesArray.length === array.length;
 }
```

Comment:
Very complicated logic.
You need to create `for in` loop for robot object and inside loop you can check if key exists in object `inversed` or not. If exist then return `null` if not add this key to `inversed` object